### ISSUE TITLE
Question: changing reaction time

### ISSUE THREAD

**Issue by wjquigs11** [mairas/ReactESP#26]
Am I correct to assume that if I configure a reaction with:
RepeatReaction app.onRepeat(uint32_t t, react_callback cb);
The time (t) is fixed, and the only way to change the repeat time is to delete the reaction and recreate?

**Comment by mairas** [comment]
Yes, at the moment that is the case. Due to the way std::priority_queues, the data structure that keeps track of sorted TimedReactions, work, changing the time would have to be done basically by removing and reinserting. Setting the future interval, however, would be trivial. I don't have time to do it myself but would be happy to review and merge a PR that would do that.
